terraform/deployments/govuk-publishing-infrastructure/app_search_s3.tf

```diff
@@ -0,0 +1,13 @@
+# TODO: instead of granting write access to nodes, use IRSA (IAM Roles for
+# Service Accounts aka pod identity) so that only search-api containers can write.
+resource "aws_iam_role_policy_attachment" "search_relevancy_s3_eks_policy_attachment" {
+  role       = data.terraform_remote_state.cluster_infrastructure.outputs.worker_iam_role_name
+  policy_arn = data.terraform_remote_state.app_search.outputs.search_relevancy_s3_policy_arn
+}
+
+# TODO: instead of granting write access to nodes, use IRSA (IAM Roles for
+# Service Accounts aka pod identity) so that only search-api containers can write.
```

Comment:
trivial suggestion: no need to duplicate this TODO; it's clear that it applies to both.
```suggestion
```

Reply:
thanks, I will fix that.